domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -28,6 +28,7 @@ enum class ReviewScheduleCalculationRule {
   LESS_THAN_6_MONTHS_TO_SERVE,
   BETWEEN_6_AND_12_MONTHS_TO_SERVE,
   BETWEEN_12_AND_60_MONTHS_TO_SERVE,
+  MORE_THAN_60_MONTHS_TO_SERVE,
```

Comment:
I'd forgotten this calculation rule previously. I only realised it was missing once I got into the implementation based on the business rules docs (see impl in `ReviewService` below)